test/grouped_gemm/test_grouped_gemm_util.hpp

```diff
@@ -50,23 +49,77 @@ class TestGroupedGemm : public testing::TestWithParam<int>
     using BDataType = std::tuple_element_t<4, Tuple>;
     using EDataType = std::tuple_element_t<5, Tuple>;
 
+    using Row = ck::tensor_layout::gemm::RowMajor;
+    using Col = ck::tensor_layout::gemm::ColumnMajor;
+
     public:
     static constexpr bool verify_     = true;
-    static constexpr int init_method_ = 1; // decimal value initialization
+    static constexpr int init_method_ = 2; // integer value initialization
```

Comment:
Why init_method = 2? Integer values should be good for tests

Reply:
Hmm good question... :D Will revert change.